Dialogue/co_040127.csv

```diff
@@ -1,7 +1,7 @@
 Title#0,"""『ランクアップとバトルコイン』について"""
 text0001#0,"""バトル終了時、勝敗に応じて<br>アリーナポイントが変動します。<br><br>アリーナポイントは勝ち続けるほど増えていき<br>自身の所属するランクが上がっていきます。<br><br><c ffff00>高いランクになるほど<br>貴重なアイテムと交換できるバトルコインが<br>多くもらえるようになります。<c>"""
 text0002#0,"""バトルアリーナで手に入れたバトルコインは<br><c ffff00>所属シップのショップエリア２Ｆの<br>バトルコイン交換ショップにて<br>様々なアイテムと交換することができます。<c><br><br>バトルアリーナで勝ち残り続けて<br>より高ランクを目指していきましょう。<br><c ffff00>\u3000※ランクの詳細はバトルカウンターの<br>\u3000\u3000\u3000「バトルアリーナの説明を聞く」から<br>\u3000\u3000\u3000確認することができます。<c>"""
-name01#0,"""ラベルナ"""
+name01#0,"""Lavernat """
```

Comment:
Undo this

Reply:
if I put """Lavernat""" in the future it gives me error when you rename